src/wizard/Step_1/components/ViewPortfolio.vue

```diff
@@ -1,11 +1,149 @@
-<!--<template></template>-->
+<template>
+  <div class="view-portfolio body-lg">
+    <v-row>
+      <v-col cols="12" offset="1">
+        <h1 class="mt-5 mb-3 h1 font-weight-bold">My Porfolios WIP</h1>
+      </v-col>
+    </v-row>
+    <div class="portfolio-banner">
+      <v-row class="d-flex">
+        <v-col class="h3 font-weight-bold align-self-center" offset="1"
+          >My Portfolios</v-col
+        >
+        <v-col class="d-flex justify-space-around">
+          <v-btn class="banner-btn" color="primary" to="#">
+            Create New Portfolio</v-btn
+          >
+        </v-col>
+      </v-row>
+    </div>
+    <v-row>
+      <v-col offset="1">
+        <v-card width="30rem" class="mt-5 flex-nowrap">
+          <div class="d-flex flex-nowrap align-center">
+            <v-card-title class="portfolio-name h3 font-weight-bold pb-0 col-11"
+              >Portfolio ></v-card-title
+            >
+            <div class="draft d-flex align-center ml-auto">draft</div>
+          </div>
+          <div>
+            <v-card-subtitle class="body pt-0">
+              This Portfolio will be used to build,test and manage the native
+              applications for the defense logistics agency.
+            </v-card-subtitle>
+          </div>
+          <div class="d-flex portfolio-info pa-0 ml-5">
+            <v-card-text class="body pa-0 ml-3">portfolio managers</v-card-text>
+            <v-card-text class="body col-1 pa-0 mr-2">2</v-card-text>
+          </div>
+          <v-divider></v-divider>
+          <v-card-actions class="d-flex">
+            <a class="h6 text-decoration-underline mt-1 mx-1" href="#">Open</a>
+            <a class="h6 text-decoration-underline mt-1 mx-1 ml-auto" href="#"
+              >Delete</a
+            >
+          </v-card-actions>
+        </v-card>
+      </v-col>
+    </v-row>
+  </div>
+</template>
 
 <script lang="ts">
 import Vue from "vue";
 import { Component } from "vue-property-decorator";
+export interface Portfolio {
```

Comment:
Great job using an interface!  I love Typescript!!!!

Let's remove the interfaces from the components and place them in a file where they can be accessed globally.  Here's how to do that....

1. Create a folder named 'types' in the root of application.
2. Create a file named Portfolio.ts and in the 'types' folder.
3. Copy paste your interfaces into the Portfolio.ts file.
4. Add a reference to the Porfolio.ts file just below this line `import { Component } from "vue-property-decorator";`  YOu have to add the interfaces you want to you use so your code will look like this... `import { Portfolio, Application, Environments } from "../../types/NavItem" `